config/database.template.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
It looks like this file is a duplicate of `config/templatest/database.template.yml`.  Okay to remove?

Reply:
Absolutely! Good catch!